src/apphosting/secrets/index.ts

```diff
@@ -215,25 +215,33 @@ export async function fetchSecrets(
  * to ensure the exported secrets accurately reflect the chosen environment.
  *
  * @param allAppHostingYamlPaths An array of paths to all available `apphosting.*.yaml` files.
- * @returns A Promise that resolves to an `AppHostingYamlConfig` object representing the combined
```

Comment:
To be clear, you can still say

@returns the final merged config to export secrets from

I was mainly suggesting you remove the type information